modal/_utils/blob_utils.py

```diff
@@ -43,6 +54,9 @@
 # TODO(dano): remove this once we stop requiring md5 for blobs
 MULTIPART_UPLOAD_THRESHOLD = 1024**3
 
+# For block based storage like volumefs2: the size of a block
+BLOCK_SIZE: int = 8 * 1024 * 1024
```

Comment:
Do we want this to be determined by the client or the server? What happens if a user disregards this value?

Reply:
We have equivalent safety checks on the server side, so you will get "bad request" type error messages.